skill-tree /src/main/java/com/RDS/skilltree/Endorsement/EndorsementController.java

```diff
@@ -0,0 +1,25 @@
+package com.RDS.skilltree.Endorsement;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("${API_V1_PREFIX}/endorsements")
+public class EndorsementController {
+
+    private final EndorsementService endorsementService;
+
+    @Autowired
+    public EndorsementController(EndorsementService endorsementService) {
+        this.endorsementService = endorsementService;
+    }
+
+    @GetMapping()
```

Comment:
You can mention the path as "/" it will work in both cases though

Reply:
Any reason for mentioning it here? Asking this as its mentioned in RequestMapping so

Reply:
people i've seen usually mention with the API path, rather than the base URL